packages/conf-ncurses/conf-ncurses.1/opam

```diff
@@ -7,6 +7,9 @@ license: "MIT"
 build: [
   ["pkg-config" "ncurses"] { os != "darwin"}
 ]
+depends: [
+  "conf-pkg-config" { os != "darwin"}
```

Comment:
Do we need a `conf-osx` package so that you can use something like `("conf-pkg-config" | "conf-osx")`?